Controller/ProfilerController.php

```diff
@@ -54,8 +55,19 @@ public function explainAction($token, $connectionName, $query)
         /** @var $connection \Doctrine\DBAL\Connection */
         $connection = $this->container->get('doctrine')->getConnection($connectionName);
         try {
-            $results = $connection->executeQuery('EXPLAIN '.$query['sql'], $query['params'], $query['types'])
-                ->fetchAll(\PDO::FETCH_ASSOC);
+            if ($connection->getDatabasePlatform() instanceof SQLServerPlatform) {
```

Comment:
Sure that is a minor change, but will that be enough for a merge? If it's not enough i will wait until there are more comments.